container.go

```diff
@@ -32,12 +32,6 @@ type ContainerConfig struct {
 	// RootFs is the container workload image on the host.
 	RootFs string
 
-	// Interactive specifies if the container runs in the foreground.
-	Interactive bool
-
-	// Console is a console path provided by the caller.
-	Console string
-
```

Comment:
@sboeuf I understand, thanks. I'd prefer if you create a separate PR for the tty and interactive fixes, together with an issue to track the problem.